### PR TITLE
I.MX6ULL secondary boot (part 1)

### PR DESCRIPTION
**Pull request by igoropaniuk**
Doesn't support switching to secondary boot path yet.

### CODE REVIEW

**Review by Tim-Anderson — APPROVED** [review]
LGTM

File: arch/arm/mach-imx/imx_secondary_boot.c [hunk 1]
```diff
@@ -100,3 +102,4 @@ U_BOOT_CMD(
 	"Assert WDOG Software Reset Signal (internal reset) via TF-A",
 	"\n"
 );
+
```
Comment (ricardosalveti) [inline arch/arm/mach-imx/imx_secondary_boot.c]: Cosmetic, no need for the empty line here.

**Comment by ricardosalveti** [comment]
Looks good, just one minor comment (cosmetics).

**Review by vanmaegima — APPROVED** [review]

File: arch/arm/mach-imx/imx_secondary_boot.c (hunk 1 above)
Comment (igoropaniuk) [reply arch/arm/mach-imx/imx_secondary_boot.c]: sure, will fix

**Comment by igoropaniuk** [comment]
added one additional commit

**Comment by ricardosalveti** [comment]
> added one additional commit

Missing [fio] tag on this one.

**Comment by igoropaniuk** [comment]
@ricardosalveti fixed

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**